orca-web/src/main/java/com/netflix/spinnaker/orca/controllers/ExecutionsImportController.java

```diff
@@ -70,10 +71,12 @@ ExecutionImportResponse createExecution(@RequestBody PipelineExecution execution
     try {
       application = front50Service.get(execution.getApplication());
       log.info("Importing application with name: {}", application.name);
-    } catch (RetrofitError e) {
-      if (e.getKind() == RetrofitError.Kind.HTTP && e.getResponse().getStatus() != 404) {
+    } catch (SpinnakerHttpException e) {
```

Comment:
What you have here matches the [original logic](https://github.com/spinnaker/orca/blob/7e923cc05af0f66b952245722aa69c667f685c5c/orca-web/src/main/java/com/netflix/spinnaker/orca/controllers/ExecutionsImportController.java#L73-L77).  I have to wonder if it's really correct though.  If the idea is to be silent about 404s, but warn on other things, wouldn't we want to have one `catch (SpinnakerServerException)` block that logs a warning for anything else?

Reply:
The point of this PR is to move to Spinnaker*Exception, so let's leave the logic as it is for now.